internal/supervisor/installation.go

```diff
@@ -688,7 +689,7 @@ func (s *InstallationSupervisor) deleteInstallation(installation *model.Installa
 }
 
 func (s *InstallationSupervisor) finalDeletionCleanup(installation *model.Installation, logger log.FieldLogger) string {
-	err := s.aws.DeletePublicCNAME(installation.DNS, logger)
+	err := s.aws.DeletePublicCNAME(strings.TrimSpace(installation.DNS), logger)
```

Comment:
I would like to see us not trim the value after we have stored it in the database. It should be valid before we store it.

Do we need this to cleanup a bad record in one of our accounts? I know we really try to avoid manually changing the DB, but I would prefer we do that here if we have a bad record.

Reply:
yes there is one bad thing on staging. but sounds good will refactor

Reply:
done